en/4-empower-staff-deliver-better-services.md

```diff
@@ -32,29 +32,25 @@ altLangPage: 4-permettre-personnel-offrir-meilleurs-services
 
 <section class="dpgn-section-guideline">
 
-## 4.1 Provide ongoing training and learning opportunities to continually improve the skills of the team and the broader network
+## 4.1 Build a team culture that is collaborative and adaptive, and is committed to enabling quality services for users
 
 <div class="dpgn-section-intro-guideline">
 
-**[TODO: Add/revise introductory text]**
+We must seek out and use modern methods such as Agile to ensure that the digital services we own and implement are innovative. We must ensure that our teams are consistently engaged through various opportunities to learn and participate in knowledge-sharing, and that we are successfully collaborating with both internal and external partners.
 
 </div>
 
 <section class="dpgn-section-checklist">
 
 ### Checklist
 
-**[TODO: Add/revise checklist items]**
-
-**Automated Decision Systems:**
```

Comment:
The Automated Decision Systems content is currently being used by a view. If you don't want it to be visible in the output of Standards 4 then apply <code>.dpgn-standard-hide</code> to the content rather than deleting it (so it still remains visible in the separate view).